app/i18n/de/conf.php

```diff
@@ -101,9 +101,9 @@
 	),
 	'mark_read_button' => array(
 		'_' => '„Alle als gelesen markieren“ Button',
-		'big' => 'Big',	// TODO
-		'none' => 'None',	// TODO
-		'small' => 'Small',	// TODO
+		'big' => 'Groß',
+		'none' => 'Keine',
```

Comment:
```suggestion
		'none' => 'Kein',
```

Reply:
Looks good

Reply:
Should be klein surely. ;-)

```suggestion
		'none' => 'Klein',
```

Reply:
„Klein“ means small so „kein“ for none is right.

Reply:
Er yes, I thought it said big and small.